en/docs/references/extend/host-authentication-endpoint-on-a-different-server.md

```diff
@@ -206,96 +54,90 @@ This is an additional improvement which enables hosting `accountrecoveryendpoin
-        
-7.  Add the configuration given below in the `<IS_HOME>/repository/conf/deployment.toml` file by replacing the 
-    `<WEB_SERVER_HOST>` and `<WEB_SERVER_PORT>` placeholders with appropriate values of the external server.
 
+3.  Open the `<TOMCAT_HOME>/webapps/authenticationendpoint/WEB-INF/classes/RecoveryEndpointConfig.properties` file and uncomment the following line:
+
+    ```xml
+    identity.server.service.contextURL=https://localhost:9443
     ```
-    [authentication.endpoints]
-    login_url="https://<WEB_SERVER_HOST>:<WEB_SERVER_PORT>/authenticationendpoint/login.do"
-    retry_url="https://<WEB_SERVER_HOST>:<WEB_SERVER_PORT>/authenticationendpoint/retry.do"
-    request_missing_claims_url="https://<WEB_SERVER_HOST>:<WEB_SERVER_PORT>/authenticationendpoint/claims.do"
-    
-    [oauth.endpoints]
-    oauth2_consent_page= "https://<WEB_SERVER_HOST>:<WEB_SERVER_PORT>/authenticationendpoint/oauth2_authz.do"
-    oauth2_error_page= "https://<WEB_SERVER_HOST>:<WEB_SERVER_PORT>/authenticationendpoint/oauth2_error.do"
-    oidc_consent_page= "https://<WEB_SERVER_HOST>:<WEB_SERVER_PORT>/authenticationendpoint/oauth2_consent.do"
-    oidc_logout_consent_page= "https://<WEB_SERVER_HOST>:<WEB_SERVER_PORT>/authenticationendpoint/oauth2_logout_consent.do"
-    oidc_logout_page= "https://<WEB_SERVER_HOST>:<WEB_SERVER_PORT>/authenticationendpoint/oauth2_logout.do"
-    
-    [saml.endpoints]
-    logout= "https://<WEB_SERVER_HOST>:<WEB_SERVER_PORT>/authenticationendpoint/samlsso_logout.do"
-    notification= "https://<WEB_SERVER_HOST>:<WEB_SERVER_PORT>/authenticationendpoint/samlsso_notification.do"
-    
-    [passive_sts.endpoints]
-    retry= "https://<WEB_SERVER_HOST>:<WEB_SERVER_PORT>/authenticationendpoint/retry.do"
-    ```
+            
+4.  Configure the keystores.
 
-8.  Import the public certificate of the identity server to the javaca certs (or web-server's truststore) of the JVM that the authentication endpoint is running.
+    !!! info
+        The relevant certificates should be added to the corresponding keystores to properly run the authentication portal. In this tutorial, we are hosting the portal in a local server, therefore, let's use the same keystore and truststore that is in the WSO2 IS instance for this portal.
 
-    ``` 
-    keytool -export -keystore $IS_HOME/repository/resources/security/wso2carbon.jks -alias wso2carbon -file wso2carbon.cer
-    ```
+    1.  Import the public certificate of WSO2 IS to the `javaca certs` (or web-server's truststore) of the JVM where the authentication endpoint is running.
 
-    ``` 
-    keytool -import -alias wso2carbon -keystore  $WEB_APP_TRUSTSTORE -file wso2carbon.cer
-    ```
+        !!! Info
+            Be sure to replace the following placeholders:
+            - `$IS_HOME`: The path to your WSO2 IS distribution.
+            - `$WEB_APP_TRUSTSTORE`: Go to the **authenticationendpoint** web app deployed in the Tomcat server and get the path to its trustore.
```

Comment:
Is this correct ? @Thisara-Welmilla

Reply:
As discussed, created a separate issue to resolve this improvement.: https://github.com/wso2/product-is/issues/16253